core_ui/lib/src/widgets/menu_dish_image.dart

```diff
@@ -1,22 +1,38 @@
+import 'package:cached_network_image/cached_network_image.dart';
 import 'package:flutter/material.dart';
 import 'package:core_ui/core_ui.dart';
 
 class MenuDishImage extends StatelessWidget {
   final String imagePath;
+  final double height;
 
   const MenuDishImage({
     Key? key,
+    required this.height,
     required this.imagePath,
   }) : super(key: key);
 
```

Comment:
you can write this like 
const MenuDishImage({
    required this.height,
    required this.imagePath,
    super.key,
  })